mypy/versionutil.py

```diff
@@ -0,0 +1,27 @@
+from typing import NamedTuple
+
+
+class VersionInfo(NamedTuple):
+    # UNSTABLE, subject to change
+
+    major: int
+    minor: int
+    patch: int
+    release_level: str
+    iteration: int
+    mypy_version: str
+    mypy_release_level: str
+
+    def simple_str(self) -> str:
```

Comment:
would it be better if this was `__str__`
```suggestion
    def __str__(self) -> str:
```

Reply:
`NamedTuple` already provides a specialized `__str__`, and I want to keep that.